src/obsidian_ai_hub/web/routes/agents.py

```
@@ -0,0 +1,191 @@
+    content = req.content.strip() if req.content else ""
+    if not content:
+        raise HTTPException(
+            status_code=status.HTTP_400_BAD_REQUEST,
+            detail="Message content must not be empty.",
+        )
+
+    try:
+        stream_gen = agent_service.stream_session_message(session_id, content)
+        return StreamingResponse(stream_gen, media_type="text/event-stream")
+    except FileNotFoundError as e:
+        raise HTTPException(status_code=status.HTTP_404_NOT_FOUND, detail=str(e))
```

Ruff (0.16.1)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)